pkg/model/resource/resource.go

```diff
@@ -54,13 +55,14 @@ type Resource struct {
 
 	// CRDVersion holds the CustomResourceDefinition API version used for the Resource.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
+
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the ResourceData.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
 }
 
-// GVK returns the group-version-kind information to check against tracked resources in the configuration file
-func (r *Resource) GVK() config.GVK {
-	return config.GVK{
+// ResourceData returns the group-version-kind information to check against tracked resources in the configuration file
+func (r *Resource) ResourceData() config.ResourceData {
```

Comment:
`Resource.ResourceData` seems unnecesarily verbose:
```suggestion
// Data returns the group-version-kind information to check against tracked resources in the configuration file
func (r *Resource) Data() config.ResourceData {
```